checkpoint_ngfw/plugin.spec.yaml

```diff
@@ -0,0 +1,673 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: checkpoint_ngfw
+title: Checkpoint NGFW
+description: Check Point’s Next Generation Firewalls (NGFW’s) are trusted by customers for their highest security effectiveness and their ability to keep organizations protected from sophisticated fifth generation cyber-attacks
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/checkpoint_ngfw
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- firewall
+- checkpoint
+hub_tags:
+  use_cases: [application_management, threat_detection_and_response, vulnerability_management]
+  keywords: [firewall, checkpoint, decoder, utilities]
+  features: []
+
+types:
+  domain:
+    domain-type:
+      title: "Domain-Type"
```

Comment:
```suggestion
      title: "Domain"
```
Does it bomb if you omit that from the title?

Reply:
We'll find out when I get to workflow testing. Fixed